merino/src/docs/api.rs

```diff
@@ -38,6 +38,9 @@ and headers.
   Merino will return an empty list instead of attempting to query other
   languages.
 
+- `User-Agent` - A user's device form factor, operating system, and browser/Firefox version
```

Comment:
I wasn't sure what else to say here (since we don't actually reject any user agents), but happy to expound further if you have any suggestions!

Reply:
Looks good as it is. Mostly I wanted to make sure that the API noted the use of User Agent. Also I wanted to make sure that we are keeping this doc up to date, so this was good practice.